components/canvas/webgl_thread.rs

```diff
@@ -1066,21 +1066,8 @@ impl WebGLImpl {
     fn get_tex_parameter(gl: &gl::Gl,
                        target: u32,
                        pname: u32,
-                       chan: WebGLSender<WebGLResult<WebGLParameter>> ) {
-        let result = match pname {
-            gl::TEXTURE_MAG_FILTER |
-            gl::TEXTURE_MIN_FILTER |
-            gl::TEXTURE_WRAP_S |
-            gl::TEXTURE_WRAP_T => {
-                let parameter = gl.get_tex_parameter_iv(target, pname);
-                if parameter == 0 {
```

Comment:
Note: I didn't find in the spec that it shouldn't be 0. Can we double check that?

Reply:
Can you file a separate issue for this? Let's keep this PR for the refactor I'd say, that makes it more straight-forward to land :)

Reply:
Sure!